assets/src/edit-story/components/panels/design/alignment/alignment.js

```diff
@@ -116,6 +73,23 @@ const PAGE_RECT = {
   height: PAGE_HEIGHT,
 };
 
+const AlignmentButton = styled(Button).attrs({
+  variant: BUTTON_VARIANTS.SQUARE,
+  type: BUTTON_TYPES.TERTIARY,
+  size: BUTTON_SIZES.SMALL,
+})`
+  position: relative;
+
+  :focus {
+    z-index: 1;
+  }
+`;
+
+const SpecialPanel = styled(Panel)`
```

Comment:
Maybe `StyledPanel` or `DarkPanel`? Otherwise, the other panels might get offended.

Reply:
Fixed in 3febe997d.

![I'm special](https://media1.tenor.com/images/79554672c0e8b959f54f4170482c0f90/tenor.gif?itemid=5491308)